extensions/authentication-jwt/package.json

```diff
@@ -21,27 +21,29 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@loopback/authentication": "^6.0.1",
-    "@loopback/core": "^2.9.5",
-    "@loopback/rest": "^6.2.0"
+    "@loopback/authentication": "^7.0.0",
+    "@loopback/authorization": "^0.7.0",
+    "@loopback/core": "^2.10.0",
+    "@loopback/rest": "^7.0.0"
   },
   "dependencies": {
-    "@loopback/security": "^0.2.18",
+    "@loopback/authentication": "^7.0.0",
+    "@loopback/authorization": "^0.7.0",
```

Comment:
It's suspicious to depend on `@loopback/authorization`. If there are common types, we should move/add them to `@loopback/security`.

Reply:
@raymondfeng I think he tries to implement a prototype RBAC system. Building it on top of the jwt extension is easier, but I am not sure should we have the jwt and RBAC mixed in one extension, or separate them?

Reply:
@raymondfeng in new pull request https://github.com/strongloop/loopback-next/pull/6408
looks like travis job failed because of module @loopback/authorization not exist in package.json?
so should I put @loopback/authorization to the package.json ?
You said that we should move some features to security module?
https://travis-ci.com/github/strongloop/loopback-next/jobs/387527898